### PR TITLE
Add live streaming page.

### PR DESCRIPTION
**Pull request by xeodou**
Add a page for live streaming.

### CODE REVIEW

**Comment by fraserxu** [comment]
说好的要review的呢? 😱

File: src/index.template.html [hunk 1]
```diff
@@ -35,6 +36,8 @@
   ga('create', 'UA-69615758-9', 'auto');
   ga('send', 'pageview');
 </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/5.11.4/video.js"></script>
```
Comment (fraserxu) [inline src/index.template.html:39]: why not put this on Qiniu or Upyun CDN so it gets faster page load?

File: src/Components/pages/Live.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  render() {
+    const { __, query } = this.props
+    return (
+      <div className='article'>
+        {/* video */}
+        <section id='hero'>
+          <div className='bg-container live'>
+            <div className='box'>
+              {/* masthead */}
+              <video
```
Comment (fraserxu) [inline src/Components/pages/Live.js:30]: Maybe we should add a count down like "Live stream starts in 10 days" so it's less confusing?

**Comment by xeodou** [comment]
因为你太慢了，下个pr我等你review好了

File: src/index.template.html (hunk 1 above)
Comment (xeodou) [inline src/index.template.html:39]: cdnjs的cdn 在全国范围都不错，七牛可能我还要自己付钱。

File: src/Components/pages/Live.js (hunk 2 above)
Comment (xeodou) [inline src/Components/pages/Live.js:30]: 会让 hoper 做个封面。；）

**Pull request merged**